### PR TITLE
Simplify usage

### PR DESCRIPTION
**Pull request by neoziro**
- Integrate server launch
  - **BREAKING CHANGE:** Puppeteer launch options are now specified under `launch` object
- Simplify `expect` usage
  - **BREAKING CHANGE:** `expectPage()` is replaced by `expect(page)`

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,13 +1,14 @@
 {
   "presets": [
     [
-      "env",
```
Comment (TrySound) [inline .babelrc]: Why?

Comment (neoziro) [reply .babelrc]: Just to be consistent!

Comment (TrySound) [reply .babelrc]: With what? Use just used shorthand in this file `transform-object-rest-spread`.

Comment (neoziro) [reply .babelrc]: Babel 7 is coming and will remove shorthand, just psychologic preparation.

**Pull request merged**